primefaces/src/main/resources/META-INF/resources/primefaces/tristatecheckbox/tristatecheckbox.js

```diff
@@ -108,7 +108,7 @@ PrimeFaces.widget.TriStateCheckbox = PrimeFaces.widget.BaseWidget.extend({
      * Toggles this tri state checkbox in the given direction. Moves between unchecked, half-checked, and fully-checked.
      * @param {-1 | 1} direction `-1` to move backwards through the states, `+1` to move forward through the states 
      */
-    toggle:function (direction) {
+    toggle: function (direction) {
```

Comment:
This space looks like an errant change?  Other than that looks great!

Reply:
just aligned to our defaults while looking through code and found another place